integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/message/processor/test/forwarding/StoreAndForwardWithEmptyMessageBodyTesCase.java

```diff
@@ -65,25 +65,25 @@ public void init() throws Exception {
                 "CREATE TABLE IF NOT EXISTS JDBC_MESSAGE_STORE(\n" + "indexId BIGINT(20) NOT NULL AUTO_INCREMENT,\n"
                         + "msg_id VARCHAR(200) NOT NULL ,\n" + "message BLOB NOT NULL,\n" + "PRIMARY KEY ( indexId )\n"
                         + ")");
-        logViewer = new LogViewerClient(context.getContextUrls().getBackEndUrl(), sessionCookie);
+        carbonLogReader = new CarbonLogReader();
+        carbonLogReader.start();
         super.init();
     }
 
     @Test
     public void testWithEmptyMessage() throws Exception {
-        logViewer.clearLogs();
+
         String location = getESBResourceLocation() + File.separator + "messageProcessorConfig" + File.separator
                 + "EmptyMsgBodyMessageStoreTest.xml";
         String proxyContent = FileUtils.readFileToString(new File(location));
         proxyContent = this.updateDatabaseInfo(proxyContent);
-        addMessageStore(AXIOMUtil.stringToOM(proxyContent));
-        loadESBConfigurationFromClasspath(
-                "artifacts" + File.separator + "ESB" + File.separator + "messageProcessorConfig" + File.separator
-                        + "EmptyMsgBodyTest.xml");
+
+        String storeLocation = "default/message-stores/EmptyMsgBodyMessageStoreTest.xml";
```

Comment:
@sachithKay lets move this "default" also to util since it is common for all synapse artifacts.

lets pass message-stores/EmptyMsgBodyMessageStoreTest as two different arguments.

like type and name ..it would be more readable.